lab-jacob/lib/router.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const urlParser = require('./url-parser');   // module for parsing the url
```

Comment:
It's good that you're using comments here, it helps me navigate the code more easily